backend/cli.py

```diff
@@ -29,6 +33,51 @@
 app = typer.Typer()
 
 
+@app.command()
+def imdb_ratings():
+    url = "https://datasets.imdbws.com/title.ratings.tsv.gz"
+
+    dir = "../imdb_dumps/"
+    Path(dir).mkdir(exist_ok=True)
+    res = requests.get(url, stream=True)
+    ratings = []
+
+    if res.status_code == 200:
+        # First we remove the old files
+        if os.path.isdir(dir):
+            for file in os.listdir(dir):
+                os.remove(os.path.join(dir, file))
+
+        with open(f"{dir}title.ratings.tsv.gz", "wb") as f:
+            f.write(res.raw.read())
+            print("Downloaded imdb file")
+
+        with gzip.open(dir + "title.ratings.tsv.gz", "rt") as file:
+            tsv_reader = csv.reader(file, delimiter="\t")
+
+            for _ in file:
+                rating = next(tsv_reader)
+                ratings.append(
+                    {
+                        "imdb_id": rating[0],
+                        "imdb_rating": rating[1],
+                        "rating_amount": rating[2],
+                    }
+                )
+
+        all_media = client.index("media").get_documents(
+            {"fields": ["imdb_id", "id"], "limit": 200}
+        )
+
+        for media in all_media.results:
+            for rating in ratings:
+                if rating["imdb_id"] == media.imdb_id:
+                    print(media.id)
+                    client.index("media").update_documents(
+                        [{"id": media.id, "imdb_rating": rating["imdb_rating"]}]
+                    )
```

Comment:
```suggestion
        imdb_media = []

        for media in all_media.results:
            for rating in ratings:
                if rating["imdb_id"] == media.imdb_id:
                    print(media.id)
                    imdb_media.append([{"id": media.id, "imdb_rating": rating["imdb_rating"]}])
                    
        client.index("media").update_documents(imdb_media)
```
This should run a lot faster (only 1 MeiliSearch job)

Reply:
Could also be a list comp I guess 🤷

Reply:
Done